README.md

```diff
@@ -132,6 +132,8 @@ The Chemical Element definition is used by the EN10168 schema as a property in C
 | Minimum   | The minimum if defined by the product specification  |    No     |   Yes   |
 | Maximum   | The maxium as defined by the product specification   |    No     |   Yes   |
 
+WARNING: the minimum value of 0 and maximum value of 100 is no longer enforced by the JSON schema. Please ensure that these values are not exceeded, as they will not be flagged by the schema.
+
```

Comment:
this is no longer true with the regex used for validation ?
but maybe it's worthy to mention the 4 decimals limit